packages/web/src/pages/settings-page/components/desktop/ManagerMode/AccountListItem.tsx

```diff
@@ -185,11 +217,13 @@ export const AccountListItem = ({
     <Flex
       alignItems='stretch'
       justifyContent='space-between'
-      ph='xl'
+      pl='xl'
+      pr='l'
       pv='l'
+      gap='l'
       border='default'
     >
-      <ArtistChip user={user as any} showPopover={false} />
+      <ArtistInfo user={user as any} />
```

Comment:
probably don't need the `as any` cast here anymore

Reply:
Oh good call, I'll check.